src/librustdoc/html/render.rs

```diff
@@ -1414,16 +1544,52 @@ impl Context {
             Some(i) => i,
             None => return Ok(()),
         };
+        let final_file = self.dst.join(&krate.name)
+                                 .join(&format!("all{}.html", self.shared.resource_suffix));
```

Comment:
Does `all.html` really need the resource suffix? It doesn't like something that would benefit from it.

Reply:
It's for `docs.rs`. I suppose they'll have use for it.

Reply:
@onur Can you confirm whether this new page would require a resource suffix? It would be different per-crate, a new page linked from the crate-root sidebar.

Reply:
@GuillaumeGomez only common javascript and css files requires suffix, crate specific html files don't need one.

Reply:
Ok, I'll remove it then.